src/pages/install/{installConfig.agentName}.js

```diff
@@ -55,7 +62,7 @@ const InstallPage = ({ data }) => {
   const handleAgentConfigChange = ({ name, value }) => {
     tessen.track({
       eventName: 'agentConfigFileUpdated',
-      category: `${name}AgentConfigFileUpdated`,
+      category: `${capitalize(name)}AgentConfigFileUpdated`,
```

Comment:
it won't let me add a suggestion for line 17, but we also need to pull in location 

```
const InstallPage = ({ data, location }) => {
```

so that `location.pathname` is defined on line 53 and 68 and elsewhere